README_PYBIND.txt

```diff
@@ -3,24 +3,25 @@ To install pybind11, use
 
 To test the current pybind setup, proceed as follows:
 
-$ mkdir pybind-build
-$ cd pybind-build
-$ cmake .. -DPYTHON_LIBRARY_DIR="<absolutepathtorepositoryroot>/try-pybindings" -DPYTHON_EXECUTABLE="/usr/bin/python3"
- (or wherever your python3 lives)
-$ make -j8
+$ ./build.py pybind_library
+$ cd builds/pybind_library/pybinsings
 $ make install
-$ cd ../try-pybindings
-$ ls
-pydownward.cpython-310-x86_64-linux-gnu.so  test.py
-$ cat test.py
-
-Put some output.sas file into the directory. The test only runs enforced
-hill-climbing, which has a high likelihood to NOT solve a task. It can for
-example solve the first driverlog task, so if in doubt, use that one.
-
-Enjoy:
-$ python -i test.py
-
+Put some output.sas file into the directory downward/trypybindings/
+The test only runs enforced hill-climbing, which has a high likelihood
+to NOT solve a task. It can for example solve the first driverlog task,
+so if in doubt, use that one.
+
+Run with one of the following two commands
+$ ./fast-downward.py pybindings-command-line
+$ ./fast-downward.py pybindings-dynamic-import
+
+Notes on the integration into fast-downward
+
+* The command "make install" still needs to be done manually.
+  This could be integrated into build.py.
+* The path where the library is installed is now hard coded in
+  src/pybindings/CMakeLists.txt. This could be integrated into
```

Comment:
No, I didn't. For now I took the install command away and let it only be done in the builds/python_library/bin folder and I added a comment whether it would be preferable to put it in builds/python_library/pybind_libraries. You'll see it after I push my commits again.